massa-client/src/tests/scenarios.rs

```diff
@@ -6,13 +6,10 @@ use serde::de::DeserializeOwned;
 use serial_test::serial;
 use std::{thread::JoinHandle, time::Duration};
 
-const TIMEOUT: u64 = 30;
```

Comment:
Why removing this ? It can't be useful for further tests ?

Reply:
True :)